LibationWinForms/GridEntry.cs

```diff
@@ -96,8 +142,22 @@ private set
 		public string Category { get; }
 		public string Misc { get; }
 		public string Description { get; }
-		public string DisplayTags => string.Join("\r\n", Book.UserDefinedItem.TagsEnumerated);
```

Comment:
This is an interesting one. I put each tag on its own line here with the assumption that each book will have no more than 3 or 4 tags. In this use case, it's much easier to read when they're on their own line. With your change I assume they will be space delimited instead of new line delimited. I prefer them on their own line but I'm open to conversation if there's a reason you think it should be otherwise.

Reply:
Gotcha. I'll change the getter back, but leave the setter so the editor can easily change them.